src/UXClient/Models/ChartDataOptions.ts

```diff
@@ -30,6 +30,7 @@ class ChartDataOptions {
     public positionXVariableName: string;
     public positionYVariableName: string;
     public image: string;
+    public isRawData: boolean;
```

Comment:
this isnt actually referenced anywhere, should we remove it?

Reply:
it is used on client side :)